pendulum_control/include/pendulum_control/rtt_kpsr_executor.hpp

```diff
@@ -135,9 +124,10 @@ class RttExecutor : public rclcpp::Executor
   timespec start_time_;
 
 private:
-  RCLCPP_DISABLE_COPY(RttExecutor)
+  rclcpp::Executor::SharedPtr kpsrExecutor;
```

Comment:
Rename `kpsrExecutor` to `decorableExecutor`

Reply:
Done

Reply:
Done